docs/administration/robot-accounts/_index.md

```diff
@@ -147,3 +136,130 @@ docker login <harbor_address<>
+| Update Tag Retention (tag-retention)             | 1. PUT [/retentions/{id}](https://github.com/goharbor/harbor/blob/2984c2e04b3b3194cabb44470d0e37acc4b1d5c9/api/v2.0/swagger.yaml#L4950) <br/>2. POST [/retentions/{id}/executions](https://github.com/goharbor/harbor/blob/2984c2e04b3b3194cabb44470d0e37acc4b1d5c9/api/v2.0/swagger.yaml#L5009) <br/>3 PATCH [/retentions/{id}/executions/{eid}](https://github.com/goharbor/harbor/blob/2984c2e04b3b3194cabb44470d0e37acc4b1d5c9/api/v2.0/swagger.yaml#L5093)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Delete Tag Retention (tag-retention)             | 1. DELETE [/retentions/{id}](https://github.com/goharbor/harbor/blob/2984c2e04b3b3194cabb44470d0e37acc4b1d5c9/api/v2.0/swagger.yaml#L4981)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+
+
+{{< note >}}
+Public APIs are not included in the tables above because they can be accessed by any robot account
```

Comment:
Public APIs are not included in the tables above because they can be accessed by any robot account.

Any robot account or anyone?